lte/gateway/python/magma/monitord/cpe_monitoring.py

```diff
@@ -25,73 +25,73 @@
 from magma.monitord.icmp_state import ICMPMonitoringResponse
 from orc8r.protos.common_pb2 import Void
 from prometheus_client import Histogram
-from magma.subscriberdb.sid import SIDUtils
 
 SUBSCRIBER_ICMP_LATENCY_MS = Histogram('subscriber_icmp_latency_ms',
```

Comment:
To me, ALL_CAPS should be reserved for const+immutable. This variable is not immutable, so I prefer its capitalization to be lower_snake_case